docs/setup/tomcat.md

```diff
@@ -475,37 +612,39 @@ In ```/var/lib/tomcat-geoserver0/conf/server.xml```:
 ### Start the instance
 
 ```
-sudo insserv tomcat-geoserver0
-sudo service tomcat-geoserver0 start
+sudo service tomcat9-geoserver0 start
 ```
 
-## Be careful
+### In case of troubles with the GeoServer UI
 
-Remember that the geOrchestra binaries must be built according to the tomcat configuration described above.
-By default, forking the template configuration should guarantee this.
+While testing the following setup guide, we stumbled upon a strange behaviour of the geoserver UI. If you
+are encountering some strange response from GeoServer ("400 Bad Request"), it might be due to the following [upstream
+issue](https://osgeo-org.atlassian.net/browse/GEOS-9353).
 
-Since we assume that :
- - proxy and cas are served by an http connector on localhost, port 8180
- - the geOrchestra webapps, except GeoServer, proxy and cas, are served by an http connector on port 8280
- - GeoServer is served by an http connector on port 8380
+As described in the issue, this can be solved by changing geoserver's `web.xml` file, and adding the following context variable:
 
-... you should verify that:
+```
+<context-param>
+  <param-name>GEOSERVER_CSRF_WHITELIST</param-name>
+  <param-value>georchestra.mydomain.org</param-value>
+</context-param>
+```
 
-1. your reverse proxy points to port 8180 (proxy)
-1. your GenerateConfig.groovy file correctly configures your proxy to point to the webapps, namely:
+As described in the [official GeoServer documentation](https://docs.geoserver.org/stable/en/user/security/webadmin/csrf.html), you should also be able to use
+an JAVA_OPTS environment variable instead of the previous modification.
 
-```groovy
-def proxyDefaultTarget = "http://localhost:8280"
 
-properties['proxy.mapping'] = """
-<entry key="analytics"     value="proxyDefaultTarget/analytics/" />
-<entry key="extractorapp"  value="proxyDefaultTarget/extractorapp/" />
-<entry key="geonetwork"    value="proxyDefaultTarget/geonetwork/" />
-<entry key="geoserver"     value="http://localhost:8380/geoserver/" />
-<entry key="geowebcache"   value="proxyDefaultTarget/geowebcache/" />
-<entry key="geofence"      value="proxyDefaultTarget/geofence/" />
-<entry key="header"        value="proxyDefaultTarget/header/" />
-<entry key="console"     value="proxyDefaultTarget/console/" />
-<entry key="mapfishapp"    value="proxyDefaultTarget/mapfishapp/" />
-<entry key="static"        value="proxyDefaultTarget/header/" />""".replaceAll("\n|\t","").replaceAll("proxyDefaultTarget",proxyDefaultTarget)
-```
+Of course you will have to adapt the parameter to suit your setup. Setups based on the Jetty Servlet Container do not seem to be affected.
+
+
+## Be careful
+
+Remember that the geOrchestra binaries must be built according to the tomcat configuration described above.
+By default, forking the template configuration should guarantee this.
```

Comment:
???

Reply:
That was already present in the documentation, even if it appears in the diff, I have not touched this part. We can probably get rid of the paragraph, though.